tests/gramex.yaml

```diff
@@ -1279,6 +1279,13 @@ url:
         class: NER
       xsrf_cookies: false
 
+  mlpredictor:
+    pattern: /predictor
+    handler: MLPredictor
+    kwargs:
+      config_dir: $YAMLPATH
+      url: $YAMLPATH/../testlib/iris.csv
+
```

Comment:
Apologies -- let's revert to:

```yaml
    data:
        url: ...
```

This is consistent with MLHandler.

Also, let's support an optional `columns:` parameter that specifies the columns in case we use a non-MLHandler based .pkl file